tests/cross-compile/sign-wgsl.slang

```diff
@@ -26,5 +26,6 @@ void main()
     outputBuffer[0].scalar_result = scalar_result;
     outputBuffer[0].vector_result = vector_result;
     
+    // WGSL: vec2<i32>(sign((vec2<f32>
```

Comment:
I don't think the order is correct. the scalar version should appear before the vector version. Or you need to use `WGSL-DAG: ` for order independent checking.